.github/workflows/publish-images.yaml

```diff
@@ -86,19 +101,5 @@ jobs:
           push: ${{ github.event_name != 'pull_request' }}
           tags: ${{ steps.docker_meta.outputs.tags }}
           labels: ${{ steps.docker_meta.outputs.labels }}
-          build-args: |
```

Comment:
why do we not need these args anymore?

Reply:
They're set while building the binary, straight from the environment. They're not needed for anything else.